scalajslib/test/src/mill/scalajslib/MultiModuleTests.scala

```diff
@@ -39,7 +39,7 @@ object MultiModuleTests extends TestSuite {
     prepareWorkspace()
 
     def checkOpt(optimize: Boolean) = {
-      val task = if (optimize) MultiModule.client.fullOpt else MultiModule.client.fastOpt
+      val task = if (optimize) MultiModule.client.fullLinkJS else MultiModule.client.fastLinkJS
```

Comment:
Doesn't look right. The test cases, that call `checkOpt` are named `fastOpt` and `fullOpt`.